model_info/enwiki.nettrom_wp10.md

```diff
@@ -1,85 +1,85 @@
 Model Information:
 	 - type: GradientBoosting
 	 - version: 0.8.1
-	 - params: {'random_state': None, 'learning_rate': 0.01, 'max_features': 'log2', 'scale': True, 'presort': 'auto', 'verbose': 0, 'criterion': 'friedman_mse', 'center': True, 'min_impurity_decrease': 0.0, 'multilabel': False, 'min_impurity_split': None, 'warm_start': False, 'subsample': 1.0, 'init': None, 'population_rates': None, 'max_depth': 3, 'max_leaf_nodes': None, 'loss': 'deviance', 'min_weight_fraction_leaf': 0.0, 'label_weights': None, 'n_estimators': 100, 'labels': ['Stub', 'Start', 'C', 'B', 'GA', 'FA'], 'min_samples_split': 2, 'min_samples_leaf': 1}
+	 - params: {'loss': 'deviance', 'presort': 'auto', 'labels': ['Stub', 'Start', 'C', 'B', 'GA', 'FA'], 'scale': True, 'min_samples_leaf': 1, 'min_impurity_split': None, 'criterion': 'friedman_mse', 'label_weights': None, 'learning_rate': 0.01, 'n_estimators': 100, 'warm_start': False, 'max_depth': 3, 'subsample': 1.0, 'center': True, 'min_impurity_decrease': 0.0, 'min_weight_fraction_leaf': 0.0, 'max_leaf_nodes': None, 'init': None, 'min_samples_split': 2, 'max_features': 'log2', 'population_rates': None, 'random_state': None, 'verbose': 0, 'multilabel': False}
```

Comment:
It looks like the best GB model in tuning ran with `learning_rate=0.01, n_estimators=500, max_depth=7, max_features="log2"`  It doesn't look like those are the params being reported here.  Maybe the model would work better if we use these?

Reply:
See https://github.com/wikimedia/articlequality/blob/master/Makefile#L105

Reply:
As per the call, I changed this to the best performing parameters with `n_estimators = 100`.